### PR TITLE
Plotly implementation for Time Vs Temperature

### PR DESCRIPTION
**Pull request by muneebafzal**
Changed the refresh configurations in dashboard.html. Added plotly visualization below the data table in the /dashboard/ page.

### CODE REVIEW

File: mercury/templates/dashboard.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8"  http-equiv="refresh" content="5; URL=/dashboard/"/>
+    <meta charset="UTF-8">
+{#    http-equiv="refresh" content="5; URL=/dashboard/"/>#}
```
Comment (de846) [inline mercury/templates/dashboard.html]: Just want to make sure you intended to leave this commented out.

File: mercury/templates/dashboard.html [hunk 2]
```diff
@@ -58,5 +59,46 @@ <h3>The Simulated Data for each data packet sent is shown:</h3>
         {#            Compression/Suspension of the 4 Wheels: FR: {{item.suspension_fr}}, FL: {{item.suspension_fr}}, BR: {{item.suspension_fr}}, BL: {{item.suspension_fr}}<br>#}
     </table>
 </div>
+
+	<div class="row">
+        <div id="articles" style="width:1400px;height:800px;"></div>
+    </div>
+	<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
+	<script src="http://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```
Comment (de846) [inline mercury/templates/dashboard.html]: FYI: there is a jQuery in our repo so you don't have to use the one from the internet, if you want.

**Comment by de846** [comment]
You'll need to add tests for the code you added, but I won't make it block getting this released today if you want to demo it.

**Review by de846 — APPROVED** [review]
Left comments. Looks okay to approve, but please consider the comments.

File: mercury/templates/dashboard.html (hunk 1 above)
Comment (muneebafzal) [reply mercury/templates/dashboard.html]: yes

File: mercury/templates/dashboard.html (hunk 2 above)
Comment (muneebafzal) [reply mercury/templates/dashboard.html]: Sure. For now, I will go with it. Will change it afterward

**Pull request merged**